docs/reference/announcements.md

```diff
@@ -4,6 +4,22 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
+## Camunda 8.5
+
+### Support for Amazon OpenSearch
+
+With the 8.5 release, Optimize is now also compatible with [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5+. Note that using Amazon OpenSearch requires [setting up a new Camunda installation](/self-managed/platform-deployment/overview.md). A migration from previous versions or Elasticsearch environments is currently not supported.
```

Comment:
This text suggests that a migration from Elasticsearch environments might be eventually supported, which we don't intend to do.

Reply:
@grlimacan I changed the text to:
"A migration from previous versions or Elasticsearch environments is not supported."
Does this make it more clear?

Reply:
Sounds good